pages/user.py

```diff
@@ -60,12 +60,26 @@ class NotificationsPage(BaseUserSettingsPage):
 
     identity = Locator(By.CSS_SELECTOR, '#notificationSettings')
 
+class EmberDeveloperAppsPage(BaseUserSettingsPage):
+    url = settings.OSF_HOME + '/settings/applications/'
+
+    identity = Locator(By.CSS_SELECTOR, '._Top__explanation_1k7ky0')
```

Comment:
I'm unsure if this is a unique identifier for this page.

Reply:
This is a local style and that last part will change with each build. Same with the other one. Do you need a `data-test` selector somewhere?

Reply:
I don't think those classes do change with each build (I think they're based off the name of the component). But I do think that using `data-test` or `data-analytics` attributes as selectors will be more reliable. So more test selectors would be lovely. (Info based on discussion with @aaxelb)

Reply:
Will do. I'll add them for all of the panels, and within the past day we've decided against using `data-analytics` tags as selectors because those could change for reasons unrelated to tests (i.e. product want's them to be something else). If you have anything specific aside from just wanting them for panels, let me know.

Reply:
Will do. Thank you very much!